docs/ionq/jobs.md

```diff
@@ -0,0 +1,120 @@
+# Running IonQ API Jobs
+
+Here we detail how to run jobs on the IonQ API against the IonQ QPU and the
+IonQ simulator.
+
+In this section we assume a `cirq.ionq.Service` object has been instantiated and is
+called `service` and `cirq` and `cirq.ionq` have been imported:
+```python
+import cirq
+import cirq.ionq as ionq
+service = ionq.Service()
+```
+See [IonQ API Service](service.md) for how to set the service up.
```

Comment:
`set the service up.` - > `set up the service`

Reply:
done